source/_components/switcher_kis.markdown

```diff
@@ -2,30 +2,33 @@
 layout: page
 title: "Switcher"
 description: "Controlling your Switcher V2 Water Heater."
-date: 2018-02-10 18:30
```

Comment:
It's not necessary to change this every time 😉

Reply:
LOL I guess you're right!
:-)